hack/go-version-check.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# List or check that the golang version embedded in files (specifically our Dockerfile)
+# is using a supported version of Go.
+# Run with --show to list the images the script thinks are current.
+# Run with --check (the default) to verify the Dockerfile contains a current
+# image & version tag.
+#
+
+set -e
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+PROJECT_DIR="$(cd "${SCRIPT_DIR}/.." && pwd)"
+WORKDIR="$(mktemp -d)"
```

Comment:
If I read this correctly, in case user provides `GOVERSIONSFILE` this temp-directory is created and removed upon exit without being used -- is this correct?

Reply:
Yes. I had considered trying to avoid that but it seemed like much more work for very little payoff because the downside of the extra mkdir/rmdir is negligible.

Reply:
I agree -- better to keep it as is and have simple code.